3.11.22/js/pull.js

```diff
@@ -0,0 +1,43 @@
+let userBorn = prompt('What year of your born?')
+let age = 2022 - userBorn
```

Comment:
2022 -- magic number. It will be better to create a variable or use `new Date().getFullYear()` to get up-to-date info